### ISSUE TITLE
allow for gallery navigation

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#132]
```
What steps will reproduce the problem?
1. take a photo or screenshot.
2. open the gallery and check for the photo.
3. you can't see the photo with talkback.

What is the expected output? What do you see instead?
you expect to see the photo that you just took so you can share it.
you don't see the photo instead you see the drop down of what you can see, and 
the more options button.
What version of the product are you using? On what operating system?
jellybean 4.1.1

Please provide any additional information below.
not being able to see the photos is a large loss to blind or visually impaired 
people, because they should have equal access to there devices and apps as 
their sighted peers.

```

Original issue reported on code.google.com by `jmwalto...@gmail.com` on 11 Sep 2012 at 12:10

**Comment by GoogleCodeExporter** [comment]
```
Cannot be fixed from TalkBack. Bug filed against Android for Gallery 
application.
```

Original comment by `alanv@google.com` on 13 Sep 2012 at 6:02
- Changed state: **WontFix**
